python/ray/tests/test_gcs_fault_tolerance.py

```diff
@@ -828,6 +829,37 @@ def f():
     wait_for_pid_to_exit(gcs_server_pid, 10000)
 
 
+@pytest.mark.skipif(
+    enable_external_redis(), reason="Only valid when started without external redis"
+)
```

Comment:
Should we remove this and if external redis is enabled, we just make sure the raylet is healthy?

Reply:
done